utils/validateUser.js

```diff
@@ -0,0 +1,37 @@
+const Joi = require('@hapi/joi');
+
+module.exports.validateRegisterInput = Joi.object({
```

Comment:
Simplify `module.exports.*` to just `exports.*`

Reply:
fixed